Git Beginning Course

NareshBMV opened the pull request:
Hi Sam,
It was great tutorials and I enjoyed it alot, thanks Sam and I would be starting now with Mastering Git.

sammyd committed:
Initial batch of clickbait ideas

sammyd committed:
Adding suggestions from Mic

Commit:
Adding sleep idea

Commit:
We should all sleep more

Commit:
Free pizza idea

Commit:
We need a video course on ASM

NareshBMV committed:
All changes committed

NareshBMV committed:
Moving file

NareshBMV committed:
Removing file with git command

NareshBMV committed:
merging clickbait

NareshBMV committed:
Some changes in readme

NareshBMV committed:
Adding message in readme file

NareshBMV committed:
Merge branch 'readme_improvements'
"Adding no-fastforward merge commit"

NareshBMV committed:
Merge branch 'clickbait'
"Merging clickbait form other remote branch clickbait branch"

NareshBMV committed:
Merge remote-tracking branch 'iwantmyrealname/master'

NareshBMV committed:
adding x in tutorials article ideas

NareshBMV committed:
Adding checkmark in book ideas

NareshBMV committed:
modified data in books

NareshBMV committed:
Merge pull request #1 from NareshBMV/status_update

Demo Pull Request

sammyd commented:
Hey @NareshBMV —congrats on submitting a PR at the end of the git course—or in fact 2 PRs! Thank you very much for your kind words about the course—I'm really glad you enjoyed it.

If you weren't already using git for your code, hopefully this has inspired you to do so. I hope that you enjoy the "Mastering Git" video course as well—you'll soon be able to get git to bend to your will :)

You can now wear this #GitWarrior badge with pride—and [Tweet](http://ctt.ec/3GUTW) your historic victory!

![git-badge](https://user-images.githubusercontent.com/658798/28743836-baed796c-744b-11e7-8cfe-11cec54f3d2e.png)


sam

Commit:
Adding the new tutorials by tutorials book idea

Convention dictates that you title a commit, and then miss a line, before filling in any detailed comments about the commit.

Commit:
Creating the tutorials directory

Commit:
Starting the arts'n'crafts team

Commit:
Completed the CVS book

Commit:
Adding the new book ideas

Commit:
Moving the iOS ideas file

Commit:
Removing the live streaming ideas

Commit:
Adding .gitignore

Commit:
Merge branch 'clickbait'

This incorporates the excellent new clickbait artcile ideas

Commit:
Adding request for action to the readme

Commit:
Adding a signature

Commit:
Merge branch 'readme_improvements'

Forcing a merge commit, even though one isn't necessary

Commit:
Merge branch 'clickbait'

From iwantmyrealname upstream repo

Commit:
Merge remote-tracking branch 'iwantmyrealname/master'

NareshBMV committed:
Merge remote-tracking branch 'lmw4051/master'
"Merging master from lmw4051"

NareshBMV committed:
adding new branch

NareshBMV committed:
Merge pull request #2 from NareshBMV/newBranch

adding new branch
